v-next/hardhat-keystore/src/internal/loaders/file-manager.ts

```diff
@@ -0,0 +1,26 @@
+import type { FileManager, UnencryptedKeystoreFile } from "../types.js";
```

Comment:
Can this be removed? It looks like a wrapper around a wrapper, maybe the whole logic can go inside the KeystoreFileLoader?

Reply:
This is here entirely to enable tests. It narrows the API for mocking to the narrowest point we can get away with. My intent is that by having just this one point of file mocking we can remove a fair chunk of the helpers for higher level mocking in the tests.